### DISCUSSION TITLE
How to use SSR but serving Static files from a CDN with an Express server?

### DISCUSSION THREAD

**Discussion by anatolzak** [vercel/next.js#23663] [Help]
Hi there!

I am using express.js as my backend to handle api routes as well as handling frontend next requests (like here https://github.com/vercel/next.js/blob/canary/examples/custom-server-express/server.js). 

That combination allows me to serve the api routes and frontend nextjs SSR routes from the same domain.

I was wondering if there is a way to serve the generated static files from a separate CDN so that my express server won't be busy serving static files and instead can only serve server side rendered nextjs routes and api routes.

Thanks in advance for any help!

**Comment by bacher** [comment]
I suppose this will help https://nextjs.org/docs/api-reference/next.config.js/cdn-support-with-asset-prefix
